README.md

```diff
@@ -7,48 +7,67 @@
 
 [Amazon DynamoDB](https://aws.amazon.com/dynamodb/) is a fully managed, serverless, key-value NoSQL database designed to run high-performance applications at any scale. DynamoDB offers built-in security, continuous backups, automated multi-region replication, in-memory caching, and data export tools.
 
+## Overview
+
 The connector provides the capability to programatically handle AWS DynamoDB related operations.
 
-For more information, go to the module(s).
-- [aws.dynamodb](./Module.md)
+This module supports [Amazon DynamoDB REST API 20120810](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/Welcome.html).
+
+## Setup guide
+
+### Step 1: Create an AWS Account
+* If you don't already have an AWS account, you need to create one. Go to the [AWS Management Console](https://console.aws.amazon.com/console/home), click on "Create a new AWS Account," and follow the instructions.
 
-## Set up DynamoDB credentials
+### Step 2: Get the Access Key ID and the Secret Access Key
```

Comment:
```suggestion
### Step 2: Get the access key ID and the secret access key
```

Reply:
Fixed with [74810e8](https://github.com/ballerina-platform/module-ballerinax-aws.dynamodb/pull/60/commits/74810e8a132acb9d3a000e4a78beb778a2091a97) and [e872960](https://github.com/ballerina-platform/module-ballerinax-aws.dynamodb/pull/60/commits/e872960fd7d5143261848f3298c8fe07acc889c0)